### PR TITLE
Style changes to merging

### PR DESCRIPTION
**Pull request by assertchris**

### CODE REVIEW

**Comment by assertchris** [comment]
screen shot 2015-04-28 at 10 40 57 am

**Comment by A7DC** [comment]
Can you show me what it looks like expanded?

File: scss/cms.scss [hunk 1]
```diff
@@ -153,11 +153,17 @@ label.right.toggle-description-correct-right {
 }
 
 .cms table.ss-gridfield-table tr td.MergeAction {
```
Comment (tractorcow) [inline scss/cms.scss]: This doesn't seem very scss and more like css. :D

Comment (assertchris) [inline scss/cms.scss]: Look now...

**Pull request merged**